cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -97,6 +97,8 @@ message CassandraClusterState {
     repeated CassandraNode nodes = 1;
     repeated ExecutorMetadata executorMetadata = 2;
     optional int64 lastServerLaunchTimestamp = 3;
+    repeated string replaceNodeIps = 4;
+    required int32 nodesToAcquire = 5;
```

Comment:
That's the "current" number of new nodes (executors) to acquire. It's decremented once an executor is acquired.
Initially it's equal to `numberOfNodes`. But once you call `replaceNode` is incremented.